Makefile

```diff
@@ -35,7 +35,7 @@ prisma-studio:
 	@docker compose exec backend sh -c "npm run db:studio"
 
 psql:
-	@docker compose exec postgres sh -c "su - postgres -c 'psql $(db)'"
+	@docker compose exec postgres sh -c "su - postgres -c 'psql $${db:-$(POSTGRES_DATABASE)}'"
```

Comment:
when not given db= it will use the POSTGRES_DATABASE env variable. makes it easier to just run `make psql`